src/features/GetCommands.ts

```diff
@@ -52,6 +52,15 @@ export class GetCommandsFeature implements IFeature {
             return;
         }
         this.languageClient.sendRequest(GetCommandRequestType, "").then((result) => {
+            const SidebarConfig = vscode.workspace.getConfiguration("powershell.sideBar");
+            const excludeFilter: string = SidebarConfig.CommandExplorerExcludeFilter;
+            if (excludeFilter !== "") {
+                const filters = excludeFilter.split(",");
+                filters.forEach((filter) => {
+                    result = result.filter((command) =>
+                        !command.moduleName.includes(filter));
+                });
```

Comment:
I think this forEach block can be shortened:

```ts
result.filter((command) => !filters.indexOf(command.moduleName));
```

Reply:
Almost correct. Just had to test for indexOf returning -1 since that's what it returns when there's nothing there.